planex/cmd/clone.py

```diff
@@ -76,10 +84,7 @@ def main(argv=None):
 
         if args.jenkins:
             print 'echo "Cloning %s"' % pin.url
-            print CHECKOUT_TEMPLATE.substitute(url=pin.url,
-                                               branch=pin.commitish,
-                                               checkoutdir=checkoutdir,
-                                               credentials=args.credentials)
+            clone_jenkins(pin.url, checkoutdir, pin.commitish, args.credentials)
```

Comment:
clone() isn't side effect free, so I wanted to keep a similar signature for clone_jenkins()